runtime/makelib/targets.mk.ftl

```diff
@@ -128,6 +128,7 @@ all: $(TARGETS)
 # Add OMR include paths
 #  Note assumptions about the directory layout of the OMR project.
 UMA_INCLUDES+=$(UMA_PATH_TO_ROOT)nls
+UMA_INCLUDES+=$(UMA_PATH_TO_ROOT)include
```

Comment:
The original PR #1902 didn't contain this change. It doesn't seem this change would be needed to add 32-bit support. Please confirm.

Reply:
This is part of mk/config script changes (no need to create a separate PR in the same repo). This is required by the Resource Compiler (RC) on Windows 32-bit to locate the requested header files at ./include

Reply:
Is there an associated omr PR? There is no omr 'include' directory.